hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/util/CryptoTokenHandlerTestBase.java

```diff
@@ -559,7 +563,7 @@ protected Token givenValidFungibleToken(
                 deleted,
                 TokenType.FUNGIBLE_COMMON,
                 TokenSupplyType.FINITE,
-                autoRenewAccountNumber,
+                AccountID.newBuilder().accountNum(autoRenewAccountNumber).build(),
```

Comment:
Please change `autoRenewAccountNumber` to `autoRenewId` and define that as ID

Reply:
Updated, thanks.